stories/topics/proc-aap-gcp-application-upgrade.adoc

```diff
@@ -0,0 +1,82 @@
+[id="proc-aap-gcp-application-upgrade"]
+
+You can upgrade your existing {AAPonGCP} deployment to the newer version. The upgrade process covers upgrading of automation hub and automation controller. The upgrade process takes roughly the same amount of time as a {AAPonGCP} deployment install. Follow the steps below to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonGCP} supports sequential upgrades. All upgrades should be no more than one major version behind what you are currently upgrading to. For example, in order to upgrade to {AAPonGCP} to 2.2.20230215, you must be on the first version of {AAPonGCP}.
```

Comment:
Isn't the `minor`?

- 2.2.1 example:
  - 2: `major`
  - 2: `minor`
  - 1: `patch`

Reply:
For sequential it would need to be the direct next one, in some cases that might be same major.minor with a different patch and in some cases it would be same major, different minor, different patch.